bedrock/products/templates/products/vpn/mac-download.html

```diff
@@ -0,0 +1,110 @@
+          <h1>{{ ftl('vpn-download-unable-to-download') }}</h1>
+          <p>{{ ftl('vpn-download-not-in-country') }}</p>
+        </div>
+      </header>
+    {% endif %}
+
+    {% if not block_download %}
+      <header class="platform-download-header">
+        <div class="mzp-l-content platform-download-header-wrapper">
+          <div class="vpn-logo">
+            <img src="{{ static('img/logos/vpn/vpn-dark-logo.svg') }}" alt="">
+          </div>
+          <h1>{{ ftl('vpn-mac-downloading-mozilla-vpn') }}</h1>
+          <p>{{ ftl('vpn-your-download-should-start', url=mac_download_url, id='vpn-download-link-mac' ) }}</p>
+        </div>
+      </header>
+      <section class="vpn-download-instructions mzp-l-content">
+        <ol class="mzp-l-columns mzp-t-columns-four">
+          {% call picto(
+            base_el='li',
+            title=ftl('vpn-open-the-file'),
+            image=resp_img(
+              url='img/products/vpn/download/vpn-file-mac.png',
+              optional_attributes={
+                'class': 'mzp-c-picto-image',
+                'loading': 'lazy'
+              }
+            ),
+            body=True,
+            ) %}
+            <p>{{ ftl('vpn-mac-go-to-your-downloads') }}</p>
+          {% endcall %}
+          {% call picto(
+            base_el='li',
+            title=ftl('vpn-launch-the-app'),
+            image=resp_img(
+              url='img/products/vpn/download/mac-installer.png',
+              optional_attributes={
+                'class': 'mzp-c-picto-image',
+                'loading': 'lazy'
+              }
+            ),
+            body=True,
+            ) %}
+            <p>{{ ftl('vpn-mac-follow-the-steps') }}</p>
+          {% endcall %}
+          {% call picto(
+            base_el='li',
+            title=ftl('vpn-login-or-signup'),
+            image=resp_img(
+              url='img/products/vpn/download/vpn-app.png',
+              optional_attributes={
+                'class': 'mzp-c-picto-image',
+                'loading': 'lazy'
+              }
+            ),
+            body=True,
+            ) %}
+            <p>{{ ftl('vpn-enter-your-email') }}</p>
+          {% endcall %}
+          {% call picto(
+            base_el='li',
+            title=ftl('vpn-turn-on-vpn'),
+            image=resp_img(
+              url='img/products/vpn/download/vpn-app-on.png',
+              optional_attributes={
+                'class': 'mzp-c-picto-image',
+                'loading': 'lazy'
+              }
+            ),
+            body=True,
+            ) %}
+            <p>{{ ftl('vpn-pick-a-recommended') }}</p>
+          {% endcall %}
+        </ol>
+      </section>
+    </div>
+    {% include 'products/vpn/includes/download-faq.html' %}
+  {% endif %}
+  <footer class="vpn-footer">
+      {% include 'products/vpn/includes/footer-legal.html' %}
+  </footer>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('vpn_platform_download') }}
```

Comment:
Issue: the reason why you get redirected to a 404 when we block downloads is because this JS file still tries to run (but finds no download and redirects you to `undefined`. Suggest adding a conditional round the bundle:
```suggestion
  {% if not block_download %}
    {{ js_bundle('vpn_platform_download') }}
  {% endif %
```